Modules/Loadable/SubjectHierarchy/Widgets/qSlicerSubjectHierarchyAbstractPlugin.cxx

```diff
@@ -284,6 +284,31 @@ QString qSlicerSubjectHierarchyAbstractPlugin::tooltip(vtkIdType itemID) const
   return tooltipString;
 }
 
+//-----------------------------------------------------------------------------
+QString qSlicerSubjectHierarchyAbstractPlugin::tooltipWithStoragePath(vtkIdType itemID) const
+{
+  vtkMRMLSubjectHierarchyNode* shNode = qSlicerSubjectHierarchyPluginHandler::instance()->subjectHierarchyNode();
+  if (!shNode)
+  {
+    qCritical() << Q_FUNC_INFO << ": Failed to access subject hierarchy node";
+    return QString();
+  }
+
+  // If associated data node is a storable node, get the file path from the URL
+  vtkMRMLNode* dataNode = shNode->GetItemDataNode(itemID);
+  if (dataNode != nullptr)
+  {
+    vtkMRMLStorableNode* storableNode = vtkMRMLStorableNode::SafeDownCast(dataNode);
+    if (storableNode != nullptr && storableNode->GetStorageNode() != nullptr)
+    {
+      QString tooltipWithStoragePath = tr("Loaded from file: ");
```

Comment:
Sentences cannot be assembled using string append, because order of words is different in each language. So, this should be changed to something like `tr("Loaded from file: %1").arg(...)`.